doc/test_organization/managing_tests_dependencies.qbk

```diff
@@ -7,17 +7,31 @@
 
 [section:tests_dependencies Managing test dependencies]
 
-Dependencies among tests are managed by means of [link boost_test.tests_organization.decorators decorators]. 
+In general, you should make sure that test cases are independent on one another.
+For any two test cases `TA` and `TB`, `TB` must not take for granted that `TA` has already executed,
+even if `TA` is declared before `TB` in the same translation unit.
+
+The only ordering-related guarantee that __UTF__ makes by default is that if test cases `TA` and `TB`
```

Comment:
I would just say that there is no order guarantied bu the framework/test runner, except otherwise specified by the dependency declaration